### PR TITLE
Big drastic change in frontend layout

### PR DESCRIPTION
**Pull request by MariamM110**
- Made the font a common theme throughout 
- added some testing
- made a few styling changes

### CODE REVIEW

File: src/Components/Account.jsx [hunk 1]
```diff
@@ -56,10 +55,13 @@ export default function Account({
     if (globalList.length > 0 && matchedUser > 0) {
       const findRank = matchedUser[0]["rank"];
       setRank(findRank);
+      console.log(matchedUser);
```
Comment (yuna-iwata) [inline src/Components/Account.jsx:58]: remove

File: src/Components/Account.jsx [hunk 2]
```diff
@@ -56,10 +55,13 @@ export default function Account({
...
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [globalList]);
 
+  console.log(globalList);
```
Comment (yuna-iwata) [inline src/Components/Account.jsx:63]: remove

**Review by yuna-iwata — COMMENTED** [review]
been counting down the seconds for this update 🦖 here you dropped this 👑

**Review by louisglan — APPROVED** [review]
Looks great!

**Pull request merged**